### PR TITLE
Add Homework9

### PR DESCRIPTION
**Pull request by ritabunets**
Be patient Yauheni 
And good luck)

### CODE REVIEW

File: Homework9_LinkedLists/LinkedListMethods.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            while (currentNode != null)
+            {
+                Console.WriteLine(currentNode.Value);
+                currentNode = currentNode.Next;
+            }
+        }
+
+        public static void GetJoinedLists(LinkedList<int> linkedNumbers1, LinkedList<int> linkedNumbers2)
+        {
+            var currentNode1 = linkedNumbers1.First;
```
Comment (yauhenists) [inline Homework9_LinkedLists/LinkedListMethods.cs]: Local variable is never used

File: Homework9_LinkedLists/LinkedListMethods.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            foreach (int linkedNumber2 in linkedNumbers2)
+            {
+                foreach (int linkedNumber1 in linkedNumbers1)
+                {
+                   if (linkedNumber1 == linkedNumber2)
+                    {
+                        Console.WriteLine(linkedNumber1);
```
Comment (yauhenists) [inline Homework9_LinkedLists/LinkedListMethods.cs]: Could you please also create a new list with with data (common values of 2 initial lists) and then writeline values for the result list

File: Homework9_QueueAndStack/QueueAndStackMethods.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
+﻿using System.Xml.Linq;
```
Comment (yauhenists) [inline Homework9_QueueAndStack/QueueAndStackMethods.cs]: Please remove unnecessary using

File: Homework9_QueueAndStack/QueueAndStackTasks.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
+﻿namespace Homework9_QueueAndStack
+{
+    class QueueAndStackTasks : QueueAndStackMethods
+    {
+        private const int ListLength = 7;
+        private const int NumberOfChars = 3;
+
+        private static void Main(string[] args)
```
Comment (yauhenists) [inline Homework9_QueueAndStack/QueueAndStackTasks.cs]: Common for all files. Main method should be public, other methods can be private

File: Homewrok9_Dictionary/City.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Reflection;
+
+namespace Homework9_Dictionary
+{
+    public class City
+    {
+        public int population;
```
Comment (yauhenists) [inline Homewrok9_Dictionary/City.cs]: Make them better as properties
public int Population { get; set; };
etc.

File: Homewrok9_Dictionary/City.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Reflection;
```
Comment (yauhenists) [inline Homewrok9_Dictionary/City.cs]: Not required using

File: Homewrok9_Dictionary/DictionaryAndLINQTasks.cs [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+                Console.WriteLine(city.Key);
+
+            Console.WriteLine("Cities ordered by population DESC:");
+            var sortedByPopulationDESC = country.OrderByDescending(c => c.Value.population).ToList();
+            foreach (var city in sortedByPopulationDESC)
+                Console.WriteLine(city.Key);
+
+            Console.WriteLine("Population of all cities:");
+            int countryPopulation = 0;
+            foreach (string key in country.Keys)
```
Comment (yauhenists) [inline Homewrok9_Dictionary/DictionaryAndLINQTasks.cs]: All is ok, not required for fixes. JFYI another simple way with linq
var sum = country.Sum(x => x.Value.population);

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Homework9_LinkedLists/LinkedListMethods.cs (hunk 1 above)
Comment (ritabunets) [reply Homework9_LinkedLists/LinkedListMethods.cs]: removed

File: Homework9_LinkedLists/LinkedListMethods.cs (hunk 2 above)
Comment (ritabunets) [reply Homework9_LinkedLists/LinkedListMethods.cs]: added

File: Homework9_QueueAndStack/QueueAndStackMethods.cs (hunk 3 above)
Comment (ritabunets) [reply Homework9_QueueAndStack/QueueAndStackMethods.cs]: easy

File: Homework9_QueueAndStack/QueueAndStackTasks.cs (hunk 4 above)
Comment (ritabunets) [reply Homework9_QueueAndStack/QueueAndStackTasks.cs]: fixed

File: Homewrok9_Dictionary/City.cs (hunk 6 above)
Comment (ritabunets) [reply Homewrok9_Dictionary/City.cs]: removed

File: Homewrok9_Dictionary/DictionaryAndLINQTasks.cs (hunk 7 above)
Comment (ritabunets) [reply Homewrok9_Dictionary/DictionaryAndLINQTasks.cs]: oh, yes, thx! updated

File: Homewrok9_Dictionary/City.cs (hunk 5 above)
Comment (ritabunets) [reply Homewrok9_Dictionary/City.cs]: updated

**Review by yauhenists — APPROVED** [review]

**Pull request merged**